backend/apps/volontulo/urls.py

```diff
@@ -50,7 +50,21 @@
         api_views.password_reset_confirm,
         name='password_reset_confirm'
     ),
-    url(r'^api/messages/$', api_views.messages_view, name='messages'),
+    url(
+        r'^api/messages/$',
+        api_views.messages_view,
+        name='messages'
+    ),
+    url(
+        r'^api/register',
+        api_views.register_view,
+        name='api_register'
+    ),
+    url(
+        r'^api/activate',
+        api_views.activate,
```

Comment:
above You have `register_view` - can we rename this one to `activate_view` instead of simple `activate`?